spec/lib/spree/shipment_notice_spec.rb

```diff
@@ -3,18 +3,87 @@
 include Spree
 
 describe Spree::ShipmentNotice do
-  let(:order_number) { 'S12345' }
-  let(:tracking_number) { '1Z1231234' }
-  let(:notice) do
-    ShipmentNotice.new(order_number:    order_number,
+
+  def define_shipment_notice(order, tracking_number = '1Z1231234')
+    ShipmentNotice.new(order_number: order.shipments.first.number,
                        tracking_number: tracking_number)
   end
 
+  context 'capture at notification is true' do
+    before do
+      Spree::Config.shipstation_capture_at_notification = true
+    end
+
+    context 'successful capture' do
+      it 'payments are completed' do
+        order = FactoryGirl.create(:completed_order_with_pending_payment)
```

Comment:
FactoryGirl syntax methods are included, so here you can just: `create(:completed_order_with_pending_payment)`